src/home/components/PetRegisterPage/index.scss

```diff
@@ -0,0 +1,58 @@
+@use '~styles/colors.scss' as colors;
+@use '~styles/breakpoints.scss' as breakpoints;
+
+.pet-regirested-page {
+  max-height: 50rem;
+  overflow: auto;
+
+  display: flex;
+
+  flex-wrap: wrap;
+
+  gap: 1.6rem;
+
+  align-items: start;
+  justify-content: center;
+
+  &__pet-card {
+    aspect-ratio: auto;
+
+    cursor: pointer;
+  }
+}
+
+.pet-regirested-footer {
+  display: flex;
+
+  margin: 2rem;
+
+  background-color: colors.$appContentColor;
+
+  &__button {
+    width: 44rem;
```

Comment:
Troca pra `max-width`, nesse caso queremos que nosso botão diminua quando a tela for menor, assim ele fluirá melhor se não definirmos uma largura chumbada